tests/integration/__init__.py

```diff
@@ -30,6 +30,11 @@
 except ImportError:
     pass
 
+try:
+    import salt.ext.six.moves.socketserver as socketserver
+except ImportError:
+    import socketserver
```

Comment:
@cachedout This needs to be moved below the salt libs. I think it has something to do with circular imports, but I don't know enough about it to say why.